public/js/lnd-for-wp-public.js

```diff
@@ -52,7 +52,7 @@ $( window ).load(function() {
 
 function poll_invoice_funded(funded_field,invoice_field,r_hash){
 
-	//console.log('polling invoice...');
+	console.log('polling invoice...');
```

Comment:
This should be commented out or removed.

Reply:
I do that